format.c

```diff
@@ -517,6 +554,24 @@ static void fmt_msg_output_str (FILE *fp,
                                                                &tstype));
 #else
                         r = fprintf(fp, "-1");
+#endif
+                        break;
+                    }
+
+                case KC_FMT_FORMATTED_TIMESTAMP:
+                {
+#if RD_KAFKA_VERSION >= 0x000902ff
+                        rd_kafka_timestamp_type_t tstype;
+
+                        char buf[80];
+                        format_timestamp(buf,
+                                         sizeof(buf),
+                                         rd_kafka_message_timestamp(rkmessage,
```

Comment:
I think we need to check that tstype is valid, on older brokers (or message format versions) the tstype will be NONE and the timestamp unusable.